views/partials/beerPartial.hbs

```diff
@@ -0,0 +1,6 @@
+{{#if image_url}}
+<img class="beer-img" src="{{image_url}}" alt="{{name}} image">
+{{/if}}
```

Comment:
Good job with the if statement here !

Reply:
Hi Anna! Thanks for the feedback! I've just finished the bonus iteration. I thought I had done it...I have committed the new changes but I'm not sure if you are able to see them in that pull request already or if I should create another pull request.

Cheers